packages/core/src/components/graphs/skeleton-lines.ts

```diff
@@ -64,30 +64,37 @@ export class SkeletonLines extends Skeleton {
 		if (isDataLoading && !isSparkline) {
 			super.renderGridSkeleton(isDataLoading);
 		} else if (isDataLoading && isSparkline) {
-			this.renderSparklineSkeleton(isDataLoading);
+			const tickValue = Tools.getProperty(
+				this.model.getOptions(),
+				"axes",
+				"left",
+				"ticks",
+				"values"
+			);
+			this.renderSparklineSkeleton(isDataLoading, tickValue);
```

Comment:
when you remove the 
```
ticks: {
    values: [16] 
}
```
you get an error because this passes in null which doesnt get overridden with the `tickValue: number[] = [100]`

this or something more elegant 
```
this.renderSparklineSkeleton(isDataLoading, tickValue ? tickValue : [100]);
```

makes sure theres a value passed in, otherwise `renderSparklineSkeleton` will also pass a null to `drawSparkline`